tests/test_smalldt.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is a fairly large/wieldy script for a unit test. It's also different in style from most other unit tests. Would it be an idea to simplify and put in one of the other unit test files?

Reply:
really ? I think it is a good, short and quick script. Also, it does replicate the error of small dt incrementals in a way that can be tested. Still, if you have a better idea I'm all ears.